web/themes/custom/novel/templates/views/node--event-list-teaser.html.twig

```diff
@@ -0,0 +1,31 @@
+<a href="{{ content.event_link }}" class="event-list-item arrow__hover--right-small">
+  <div class="event-list-item__image-container">
+    {% if content.field_event_image['#items'] is not empty %}
+      {% set image_uri = content.field_event_image['#items'].entity.field_media_image.entity.fileuri %}
+      {% set image_url = file_url(image_uri) %}
+        <img src="{{ image_url }}" alt="{{ content.label }}" class="event-list-item__image" />
+    {% endif %}
+  </div>
+  <div class="event-list-item__content">
+    <div aria-pressed="false" class="tag tag--fill cursor-pointer tag--small event-list-item__tag">foredrag</div>
+    <p class="event-list-item__date">{{ content.field_event_date|render|striptags }}</p>
+    <h3 class="event-list-item__title">{{ label }}</h3>
+    <p class="event-list-item__description">{{ content.field_event_description|render|striptags|trim }}</p>
```

Comment:
Please reconsider the `render|striptags|trim` construct.

I guess you have added this to avoid wrapper divs around the field content. I would prefer that we use a different approach:

1. Override the field template to remove the wrapper divs
2. Get the raw value from the entity